info.json

```diff
@@ -1 +1,61 @@
-{"versionFile":"espresso.php","slug":"event-espresso-core-reg","textDomain":"event_espresso","wpOrgSlug":"event-espresso-decaf","wpOrgMainFileSlug":"espresso","wpOrgUser":"eventespresso","wpOrgRelease":"4.9.60.p","wpOrgPluginName":"Event Espresso 4 Decaf","wpOrgPluginUrl":"https:\/\/eventespresso.com\/pricing\/?ee_ver=ee4&utm_source=ee4_decaf_plugin_admin&utm_medium=link&utm_campaign=wordpress_plugins_page&utm_content=support_link","name":"Event Espresso Core","jsBuildDirectory":".\/","srcBuildFolderName":"event-espresso-core","releaseFilesRemove":["src\/tests\/**","src\/acceptance_tests\/**","src\/docs\/**","src\/info.json","src\/circle.yml","src\/readme.txt","src\/wp-assets\/**","src\/.travis.yml","src\/ISSUE_TEMPLATE.md","src\/PULL_REQUEST_TEMPLATE.md","src\/assets\/src\/**"],"decafFilesRemove":["src\/tests\/**","src\/acceptance_tests\/**","src\/docs\/**","src\/info.json","src\/circle.yml","src\/.travis.yml","src\/caffeinated","src\/wp-assets\/**","src\/ISSUE_TEMPLATE.md","src\/PULL_REQUEST_TEMPLATE.md","src\/assets\/src\/**"],"github":true,"awsbucket":"","awsregion":"","remoteNamesToPushTo":["demoee","devboxsitehub"]}
\ No newline at end of file
+{
+  "versionFile": "espresso.php",
+  "slug": "event-espresso-core-reg",
+  "textDomain": "event_espresso",
+  "wpOrgSlug": "event-espresso-decaf",
+  "wpOrgMainFileSlug": "espresso",
+  "wpOrgUser": "eventespresso",
+  "wpOrgRelease": "4.9.60.p",
+  "wpOrgPluginName": "Event Espresso 4 Decaf",
+  "wpOrgPluginUrl": "https:\/\/eventespresso.com\/pricing\/?ee_ver=ee4&utm_source=ee4_decaf_plugin_admin&utm_medium=link&utm_campaign=wordpress_plugins_page&utm_content=support_link",
+  "name": "Event Espresso Core",
+  "jsBuildDirectory": ".\/",
+  "srcBuildFolderName": "event-espresso-core",
+  "releaseFilesRemove": [
+    "src\/tests\/**",
+    "src\/acceptance_tests\/**",
+    "src\/docs\/**",
+    "src\/info.json",
+    "src\/circle.yml",
+    "src\/readme.txt",
+    "src\/wp-assets\/**",
+    "src\/.travis.yml",
+    "src\/ISSUE_TEMPLATE.md",
+    "src\/PULL_REQUEST_TEMPLATE.md",
+    "src\/assets\/src\/**",
+    "src\/eslint\/**",
+    "src\/.babelrc",
+    "src\/.editorconfig",
+    "src\/.eslintignore",
+    "src\/.eslintrc.js",
+    "src\/.nvmrc",
+    "src\/.postcss.config.js"
```

Comment:
Notice I added a bunch more files/folders to the folder removal configuration (these are things that get removed on release builds)

Reply:
wait... why are we removing "readme.txt" ???
and should we be keeping ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md ?

Reply:
Eventually we should be moving the github specific md files to a `.github` folder.  That's a convention github provides to keep that sort of things organized better.  Then I just have to do the pattern against `.github`

Reply:
kk makes sense thnx